search/find-place/src/main/java/com/esri/samples/find_place/FindPlaceController.java

```diff
@@ -67,9 +68,14 @@
 
   @FXML
   public void initialize() {
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-    // create ArcGISMap with streets basemap and add it to map view
-    ArcGISMap map = new ArcGISMap(Basemap.createStreets());
+    // create ArcGISMap with streets basemap style
```

Comment:
We could plain-english this?
```suggestion
    // create a map with the streets basemap style
```